bin/metric-postgres-graphite.rb

```diff
@@ -83,6 +83,18 @@ class CheckpostgresReplicationStatus < Sensu::Plugin::Metric::CLI::Graphite
 
   include Pgpass
 
+  def compute_lag(master, slave, m_segbytes)
+    m_segment, m_offset = master.split('/')
+    s_segment, s_offset = slave.split('/')
+    ((m_segment.hex - s_segment.hex) * m_segbytes) + (m_offset.hex - s_offset.hex)
+  end
+
+  def check_vsn(conn)
```

Comment:
I feel like this is something that all the check and metric scripts would benefit from having available. Would you mind making it a library method? Also since this does not actually return the version back maybe we should call it `postgres9?` as it returns only a `true` or `false`